pkg/uki/upgrade.go

```diff
@@ -64,6 +64,17 @@ func (i *UpgradeAction) Run() (err error) {
 		return err
 	}
 
+	// Check if the upgrade artifact contains the proper signature before copying
+	err = checkArtifactSignatureIsValid(i.cfg.Fs, filepath.Join(constants.UkiEfiDir, "EFI/Kairos", fmt.Sprintf("%s.efi", UnassignedArtifactRole)), i.cfg.Logger)
```

Comment:
shouldn't this be? I guess it does not matter since it will never run on windows :P 

```suggestion
	err = checkArtifactSignatureIsValid(i.cfg.Fs, filepath.Join(constants.UkiEfiDir, "EFI", "Kairos", fmt.Sprintf("%s.efi", UnassignedArtifactRole)), i.cfg.Logger)
```